python/tester/upload_from_repo.py

```diff
@@ -309,6 +315,8 @@ def _upload_one_image(temp_dir, group_id, butler, ref):
     instrument = ref.dataId["instrument"]
     with time_this(log=_log, msg="Single-image processing", prefix=None):
         exposure_num, headers = make_exposure_id(instrument, group_id, 0)
+        if instrument == "LSSTCam-imSim":
+            headers.update(make_imsim_time_headers(EXPOSURE_INTERVAL))
```

Comment:
Same concern about passing in `now` as for the similar line in `upload_from_raws`